chalicelib/checks/wrangler_checks.py

```diff
@@ -2270,3 +2276,480 @@ def patch_hic_summary_tables(connection, **kwargs):
+                user = users[0]
+                if user['uuid'] in all_dcic_uuids:
+                    problem_msg = 'NEW OH Line for existing user, {}'.format(user['uuid'])
+                    problem.append([problem_msg, oh_mail])
+                    continue
+                updates = compare_record(a_record, user, all_labs, all_grants, new=True)
+                if updates:
+                    updates['DCIC Active/Inactive'] = '1'
+                    actions['patch_excel'][a_record['OH Account Email']] = updates
+            # time to create a new account
+            else:
+                user_data = create_user_from_oh_info(a_record, all_labs, all_grants)
+                if not user_data.get('lab'):
+                    add_awards = [i['uuid'] for i in all_grants if a_record['OH Grant'] in i['@id']]
+                    if add_awards:
+                        add_award = add_awards[0]
+                    else:
+                        add_award = a_record['OH Grant']
+                    problem.append(['cannot find the lab', a_record['OH Lab'], add_award])
+                    continue
+                actions['add_user'].append(user_data)
+
+    all_patching_uuids = [v['DCIC UUID'] for k, v in actions['patch_excel'].items() if v.get('DCIC UUID')]
+    # skip the total
+    skip_list = all_dcic_uuids + all_patching_uuids + skip_users_uuid
+    remaining_users = [i for i in fdn_users if i['uuid'] not in skip_list]
+    print(len(remaining_users), 'remaining users')
+
+    if remaining_users:
+        for a_user in remaining_users:
+            # create empty record object
+            new_record, alt_awards = generate_record(a_user, all_labs, all_grants)
+            new_record['DCIC Active/Inactive'] = '1'
+            actions['add_excel'].append(new_record)
+
+    # do we need action
+    check.summary = ""
+    for a_key in actions:
+        if actions[a_key]:
+            check.status = 'WARN'
+            check.allow_action = True
+        check.summary += '| {} {}'.format(str(len(actions[a_key])), a_key)
+
+    if problem:
+        check.status = 'WARN'
+        check.summary += '| %s problems' % (str(len(problem)))
+
+    check.full_output = {'actions': actions, 'problems': problem}
+    return check
+
+
+@action_function()
+def sync_users_oh_start(connection, **kwargs):
+    """Start runs by sending compiled input_json to run_workflow endpoint"""
+    action = ActionResult(connection, 'sync_users_oh_start')
+    my_auth = connection.ff_keys
+    sync_users_oh_check_result = action.get_associated_check_result(kwargs).get('full_output', {})
+    actions = sync_users_oh_check_result['actions']
+    user_list = get_oh_google_sheet()
+    add users
+    if actions.get('add_user'):
+        for a_user in actions['add_user']:
+            del a_user['lab_score']
+            del a_user['OH_lab']
+            a_user['viewing_groups'] = ["4DN"]
+            ff_utils.post_metadata(a_user, 'user', my_auth)
+    # delete users
```

Comment:
Again better to say remove permission from user or something rather than delete.